### ISSUE TITLE
CE v0 elements inside lit-html templates not upgrading

### ISSUE THREAD

**Issue by rictic** [Polymer/lit-html#861]
This change: https://github.com/Polymer/lit-html/commit/5840b3a5a569b729a53cbd1f6ea7526d3407ceff#diff-287797daa6e077e3d776f3d8d71dd0b3L64

Appears to have broken the use case of creating Polymer v1 (custom elements v0) elements inside of a lit-html template. The elements are rendered, but they are not upgraded.

Reverting the change to `fragment`'s initializer resolves the issue.

Issue occurs in Chrome with native CE v0, haven't yet reprod with polyfilled CE v0. External version of b/127388674

**Comment by jridgewell** [comment]
The code is using `customElements.upgrade`, so if that's not throwing you've got to have either a polyfill layer or you're running CEv1. But, Chrome 71 should be running CEv1, right?

**Comment by rictic** [comment]
Chrome has CEv1, but the app that encountered this is written with CEv0 (it's migrating to CEv1, but not finished)

**Comment by jridgewell** [comment]
I see now:

```js
document.registerElement('x-el', {
  prototype: Object.create(HTMLElement.prototype, {
    createdCallback: {
      value() {
        console.log("created");
      },
    },
  }),
});

const template = document.createElement('template');
template.innerHTML = `<x-el></x-el>`;

// This won't fire DRE upgrade.
const frag = template.content.cloneNode(true);
document.adoptNode(frag);
// Note this "upgrades" CEv1,
// but that doesn't help our DRE element.
customElements.upgrade(frag);

// Even inserting won't upgrade the DRE element
document.body.appendChild(frag);
```

**Comment by rictic** [comment]
This has been fixed at master, probably by #876 as jridgewell says

Woot!
